docker/genconfig/genconfig.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/insolar/insolar/configuration"
+)
+
+/* Return env variable or default value, it env is not presented */
+func GetEnvDefault(key, defaultVal string) string {
```

Comment:
The proper way to write comments in go is "<MethodName> does ... and returns ...". Otherwise some tools like godoc will not work properly.